cartography/intel/aws/emr.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import time
+from typing import Dict
+from typing import List
+
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+# EMR API is subject to aggressive throttling, so we need to sleep a second between each call.
+list_sleep = 1
+describe_sleep = 1
+
+
+@timeit
+@aws_handle_regions
+def get_emr_clusters(boto3_session, region) -> List[Dict]:
+    client = boto3_session.client('emr', region_name=region)
+    clusters: List[Dict] = []
+    paginator = client.get_paginator('list_clusters')
+    for page in paginator.paginate():
+        cluster = page['Clusters']
+        clusters.extend(cluster)
+        time.sleep(list_sleep)
+    return clusters
+
+
+@timeit
+def get_emr_describe_cluster(boto3_session, region, cluster_id) -> Dict:
+    client = boto3_session.client('emr', region_name=region)
+    response = client.describe_cluster(ClusterId=cluster_id)
+    return response['Cluster']
+
+
+@timeit
+def load_emr_clusters(neo4j_session, cluster_data, region, current_aws_account_id, aws_update_tag):
+    query = """
+    UNWIND {Clusters} as emr_cluster
+        MERGE (cluster:EMRCluster{id: emr_cluster.Name})
+        ON CREATE SET cluster.firstseen = timestamp(),
+            cluster.arn                 = emr_cluster.ClusterArn,
+            cluster.id                  = emr_cluster.Id,
+            cluster.name                = emr_cluster.Name,
+            cluster.servicerole         = emr_cluster.ServiceRole,
+            cluster.region              = {Region}
+        SET cluster.lastupdated         = {aws_update_tag}
+        WITH cluster
+
+        MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
+        MERGE (owner)-[r:RESOURCE]->(cluster)
+        ON CREATE SET r.firstseen       = timestamp()
+        SET r.lastupdated               = {aws_update_tag}
+    """
+
+    logger.debug("Loading EMR clusters for region '%s' into graph.", region)
```

Comment:
Let's increase this to INFO level and also log the count of clusters being loaded.